### ISSUE TITLE
SKHD and Yabai not fully working macOS Ventura 13.2.1

### ISSUE THREAD

**Issue by alanmicah** [koekeishiya/skhd#251]
I updated from macOS Monterey 12.6.3 to Ventura 13.2.1 and now skhd key bindings aren't working. Yabai still seems semi working so the bsp layout is still being applied and I can still make some changes to the yabairc however certain settings like _window_shadow_ don't seem to be working. skhd has stopped working. I have done a uninstall and new install however still no fix. I have also disabled System Integrity Protection and enabled non-Apple-signed arm64e binaries _sudo nvram boot-args=-arm64e_preview_abi_

***Update***
I'm not sure what triggered some skhd functions to work, but I can now manage and swap window positions in the same space, change focus to different windows in the same space, stack windows, and move focus between windows on the same stack. However, I cannot change spaces.

**Comment by alanmicah** [comment]
One of the errors I'm getting when using skhd --verbose is _cannot focus space due to an error with the scripting-addition._

**Comment by alanmicah** [comment]
So some steps when trying to fix, I installed command line tools again and updated Xcode, uninstalled and installed yabai and skhd from fresh, for this I followed each step in the guide on ["Installing yabai from HEAD"](https://github.com/koekeishiya/yabai/wiki/Installing-yabai-(from-HEAD)#configure-scripting-addition). I created new config executable files for both yabai and skhd. Disabled System Integrity Protection and enabled non-Apple-signed arm64e binaries sudo nvram boot-args=-arm64e_preview_abi. I had restarted my machine multiple times during this process.
